kaolin/transforms/meshfunc.py

```diff
@@ -65,8 +65,8 @@ def sample_triangle_mesh(vertices: torch.Tensor, faces: torch.Tensor,
     # We want the last dimension of vertices to be of shape 3.
     helpers._assert_shape_eq(vertices, (-1, 3), dim=-1)
 
-    dist_uni = torch.distributions.Uniform(torch.tensor([0.]).to(
-        vertices.device), torch.tensor([1.]).to(vertices.device))
+    dist_uni = torch.distributions.Uniform(torch.zeros((1,), device=vertices.device),
```

Comment:
Unrelated to PR, so suggest removing. What do you feel?

Reply:
It's (more or less) part of the speedup / clean, it avoid one unecessary transfer to GPU.